lib/railway_ipc/incoming_message.rb

```diff
@@ -47,8 +47,8 @@ def stringify_errors
 
     def get_decoder(name)
       {
-        'protobuf_binary' => RailwayIpc::MessageDecoders::ProtobufBinaryDecoder,
-        'protobuf_json' => RailwayIpc::MessageDecoders::ProtobufJsonDecoder
+        'binary_protobuf' => RailwayIpc::MessageDecoders::ProtobufBinaryDecoder,
+        'json_protobuf' => RailwayIpc::MessageDecoders::ProtobufJsonDecoder
```

Comment:
Because the keys are being changed here, is there any chance that a message being sent is using `protobuf_json` but then it gets to an app using the new gem looking for `json_protobuf`? That looks like it would default us to the binary one then. Or have these key names not been released into the wild yet?
If they are live, we may want to keep all 4 names for a while and then deprecte the old ones at a later date.

Reply:
Yeah, nothing is using these key names yet, so this is our one chance to change them w/o breaking anything. But yes, once these are being used we'd have to go the deprecation route if we want to change them.

Reply:
Ok, cool. This comment is the one I was most worried about.